pkg/scheduler/schedule_one.go

```diff
@@ -711,40 +711,31 @@ func prioritizeNodes(
 
 	if len(extenders) != 0 && nodes != nil {
 		var mu sync.Mutex
-		var wg sync.WaitGroup
 		combinedScores := make(map[string]int64, len(nodes))
-		for i := range extenders {
-			if !extenders[i].IsInterested(pod) {
-				continue
+		fwk.Parallelizer().Until(ctx, len(extenders), func(extIndex int) {
+			metrics.SchedulerGoroutines.WithLabelValues(metrics.PrioritizingExtender).Inc()
+			defer metrics.SchedulerGoroutines.WithLabelValues(metrics.PrioritizingExtender).Dec()
```

Comment:
maybe we can add some comments somewhere to define a timeline to remove this metric, fine as is.

Reply:
`Parallelizer` only has `Goroutine`, but this is `SchedulerGoroutines` besides the `Goroutine`, `SchedulerGoroutines` should be removed in a future version.

Reply:
> SchedulerGoroutines should be removed in a future version.

Yes. 

> we can add some comments somewhere to define a timeline to remove this metric

👍 
I guess we can remove it next release..? (I don't know about the deprecated process of metrics)
Let me discuss in https://github.com/kubernetes/kubernetes/issues/112034